constructor/winexe.py

```diff
@@ -342,7 +344,7 @@ def verify_signtool_is_available(info):
     if not info.get("signing_certificate"):
         return
     signtool = os.environ.get("CONSTRUCTOR_SIGNTOOL_PATH", "signtool")
-    print(f"Checking for '{signtool}'...")
+    logger.info(f"Checking for '{signtool}'...")
```

Comment:
%-format

Reply:
Updated the PR description with some reasoning on this topic.